staging/src/k8s.io/sample-controller/controller.go

```diff
@@ -61,6 +61,8 @@ const (
 	// MessageResourceSynced is the message used for an Event fired when a Foo
 	// is synced successfully
 	MessageResourceSynced = "Foo synced successfully"
+	// FieldManager distinguishes this controller from other things writing to API objects
+	FieldManager = "Foo-controller"
```

Comment:
Should this be `foo-controller`? (aka all lower case?)

Reply:
Is this a documented norm or requirement? If so, where?

Reply:
Not exactly documented, but our first 40 or so are all lower-case: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/names/controller_names.go#L44-L86

Reply:
Thanks for the pointer. I will change this one to "foo-controller".

Reply:
✅